app-electronics/nextpnr/spec

```diff
@@ -1,7 +1,7 @@
-VER=0+git20211107
-REL=4
-SRCS="git::commit=b7207b088539e7b5f34ebbd9f53f9fa14bef6e62;rename=nextpnr::https://github.com/YosysHQ/nextpnr.git \
-      git::commit=7c8ba15beb174d8def021bba6e1db06426e318a9;rename=mistral::https://github.com/Ravenslofty/mistral"
+VER=0.6
+# NOTE: nextpnr requires a specific version of mistral
```

Comment:
Please add a comment on how to track mistral versions. Also, consider adding mistral version to the main VER.

Reply:
> Please add a comment on how to track mistral versions. Also, consider adding mistral version to the main VER.

I don't know how to track mistral versions. This is purely trial and error.